lib/service/Controller/Scout/LSPController.dart

```diff
@@ -46,24 +43,14 @@ class LSPController extends GetxController {
   }
 
   Future<void> approve() async {
-    BigInt amount =  BigInt.from(createAmt.value) * BigInt.parse('250000000000000000');
     print("[Console] Inside approve()");
-    EthereumAddress address =
-        EthereumAddress.fromHex("0x76d9a6e4cdefc840a47069b71824ad8ff4819e85");
-    final tokenClient =
-        Web3Client("https://matic-mumbai.chainstacklabs.com", Client());
-    Erc20 axt = Erc20(address: address, client: tokenClient);
     try {
```

Comment:
what's the reason this was deleted?

Reply:
i readded this and some will continue this work

Reply:
thank you